tee-worker/litentry/core/credentials/src/jsonschema.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020-2023 Trust Computing GmbH.
+// This file is part of Litentry.
+//
+// Litentry is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Litentry is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Litentry. If not, see <https://www.gnu.org/licenses/>.
+
+use crate::achainable;
+use litentry_primitives::{AchainableParams, Assertion};
+use std::string::String;
+
+const BASE_URL: &str = "https://raw.githubusercontent.com/litentry/vc-jsonschema/main/dist/schemas";
+const NOT_IMPLEMENTED: &str =
+	"https://raw.githubusercontent.com/litentry/vc-jsonschema/main/dist/schemas/0-base.json";
+
+/// Returns the respective JSON Schema for the given assertion and its credential.
+/// JSON Schemas can be found on https://github.com/litentry/vc-jsonschema
+pub fn get_json_schema_url(assertion: Assertion) -> String {
```

Comment:
We could use `&Assertion` here to avoid extra copy

Reply:
Thanks– I'm quite a novice in Rust!